safetab/find_save_tools.py

```diff
@@ -33,6 +33,7 @@ def make_folders(table_config_json, path=None):
         for folder_name, table_dets in folder_names.items():
             os.mkdir(folder_name)
     else:
+        os.mkdir(path)
```

Comment:
Maybe add `(exist_ok=True)` in case the dir already exist?

Reply:
done - thanks for the tip